<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAddIdentification</class>
 <widget class="QDialog" name="DialogAddIdentification">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>266</width>
    <height>145</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddIdentify</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="labelKey1">
     <property name="text">
      <string>Key1</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QLineEdit" name="lineEditKey1"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelKey2">
     <property name="text">
      <string>Key2</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QLineEdit" name="lineEditKey2"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelHandle">
     <property name="text">
      <string>Handle</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QComboBox" name="comboBoxHandle">
     <item>
      <property name="text">
       <string>Caution</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Allow</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Refuse</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>41</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="btnOK">
     <property name="text">
      <string>OK</string>
     </property>
     <property name="qssGroup" stdset="0">
      <string>ToolButton</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="btnCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
     <property name="qssGroup" stdset="0">
      <string>ToolButton</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QLineEdit" name="lineEditKey3"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelKey3">
     <property name="text">
      <string>Key3</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
